pages/osx/readlink.md

```diff
@@ -0,0 +1,8 @@
+# readlink
+
+> Follow symlinks and get symlink information.
+> More information: <https://www.gnu.org/software/coreutils/readlink>.
+
+- Print the actual full path which the symlink points to:
```

Comment:
```suggestion
- Print the absolute path which the symlink points to:
```

Just my opinion but could we use the word absolute instead of full?
Paths are relative or absolute, while "full" is recognized/valid, it's less common.

https://en.wikipedia.org/wiki/Path_(computing)#Absolute_and_relative_paths

I also feel the use of the word "actual" is redundant here. Usually, when the word "actual" or "actually" like this, it's to add emphasis, but that's not needed here.

Any other opinions?

Reply:
sounds better to me, I'll replace it.